packages/backend/src/telemetry/service.ts

```diff
@@ -13,6 +13,7 @@ import { BaseService } from '../shared/baseService'
 export interface TelemetryService {
   shutdown(): void
   getOrCreateMetric(name: string, options?: MetricOptions): Counter
+  getOrCreateHistogramMetric(name: string, options?: MetricOptions): Histogram
```

Comment:
This doesn't have to be in this PR of course, but what do you think about changing the `TelemetryService` interface/implementation?

Looking at `getOrCreateMetric` I'm not sure if I like how some telemetry "details" are always needed in the calling services. For example, when we increment the counter, we pass in (we should always pass in) the counter attributes object: 
{
    source: deps.telemetry.getInstanceName()
} 
even though IMO, it shouldn't be the problem of the calling service to do it (even though, of course, it can provide additional attributes to the counter, if it wants).

Ideally, including the addition of the histogram, we should only have three public methods in the service: `shutdown` `incrementCounter` `recordHistogram`.

Reply:
Of course, those are just suggested methods, maybe we could also have `incrementAmountWithPrivacy` or something as well, and the `privacy` obj doesn't need to be exposed outside of `TelemetryService` itself